pkg/util/flags/flags.go

```diff
@@ -50,20 +55,19 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
-func SetLoggingFlags(flags *pflag.FlagSet) error {
+func SetLoggingFlags(flags *pflag.FlagSet) {
 	//Add commandline flags to the flagset. We will always write to stderr
 	//and not to a file by default
 	flags.AddGoFlagSet(flag.CommandLine)
 	if err := flags.Set("logtostderr", "true"); err != nil {
-		return fmt.Errorf("failed to set flag %q. %v", "logtostderr", err)
+		logger.Error("failed to restart the mon if it is stuck", err)
```

Comment:
Where did this message come from? 
```suggestion
		logger.Error("failed to set flag %q. %v", "logtostderr", err)
```